src/storage/prisma.ts

```diff
@@ -495,21 +495,16 @@ export class PrismaStorage implements IDriveStorage {
     }
 
     async deleteDocument(drive: string, id: string) {
-        await this.db.document.delete({
-            where: {
-                id_driveId: {
+        try {
+            await this.db.document.deleteMany({
+                where: {
                     driveId: drive,
                     id: id
                 }
-            },
-            include: {
-                operations: {
```

Comment:
We also need to delete operations and attachments right? Or deleteMany does this?

Reply:
Can you add an unit test to make sure everything is correctly deleted?

Reply:
the db is configured to delete also operations, attachments and syncUnits when a doc gets deleted (onCascade=delete). will provide a unit test